### DISCUSSION TITLE
Selected text not clear on Chrome

### DISCUSSION THREAD

**Discussion by hugovk** [pradyunsg/furo#441] [Potential Issue] [answered]
1. Open a Furo page, for example https://pradyunsg.me/furo/customisation/ which has a mix of normal text and code
2. Highlight all the text

With macOS Monterey:

# Chrome

The page has a dark grey background. Most of the text is selected with a black background, which is very hard to notice. I sometimes think the text is not selected at all.

(It seems a bit more noticeable in this smaller screenshot, but full size I find it harder to notice.)

The main issue is the normal text, but looking now, a secondary issue is the links and code text also have a too bright background. Some of the code is completely unreadable (`html_theme_options` in the code sample).

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/1324225/168893545-c72e9f7a-f03c-4167-b766-b67fc6abf85c.png">

# Opera

Chromium-based, same problem.

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/1324225/168894517-938c3a11-9ae1-4d4d-a171-05a37aa528ca.png">

# Firefox

Is fine, there's a light grey highlight.

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/1324225/168894012-7d213683-9529-4cf4-87c2-14f6e2909119.png">

# Safari

Also fine, with a light grey highlight. It's a slightly different shade to Firefox, so I think this is browser defaults rather than something defined in Furo.

<img width="1624" alt="image" src="https://user-images.githubusercontent.com/1324225/168894124-d1e7d33a-19d4-43ab-8ddb-dd52fc09e001.png">

**Comment by hugovk** [comment]
But Chrome on two other Macs is fine (both 101.0.4951.64, the latest):

![image](https://user-images.githubusercontent.com/1324225/169021734-e512d846-4aa8-4ed1-bd1a-9fae2c3fb1e4.png)

Also Opera is fine on these machines, with:

* Version:82.0.4227.58 (x86_64)
* Version:87.0.4390.25 (x86_64)
* Version:87.0.4390.25 (arm64)

I'll check back the versions on the first machine.

**Comment by hugovk** [comment]
Strange, the first Mac also has:

* Chrome Version 101.0.4951.64 (Official Build) (arm64)
* Opera Version:87.0.4390.25 (arm64)

**Comment by pradyunsg** [comment]
Yea, I noticed this but haven't figured out a good reproducer for this outside of "get a laptop that reproduces this".

**Comment by pradyunsg** [comment, marked as answer]
I can't reproduce this anymore, so I'm going to go ahead and close this out. This might have been resolved at some point through a change made, or I haven't reproduced this correctly.

If someone encounters this again, please file a new discussion topic with reproduction steps and include your OS and browser details.
